Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +11,335 @@ public static class BalanceStats
+        {
+            if (IsStrArrayNullOrEmpty(balance))
+            {
+                return null;
+            }
+            decimal[] auxArray = new decimal[balance.Length];
+
+            int j = 0;
+            
+            for (int i = 0; i < balance.Length; i++)
+            {
+                if (!string.IsNullOrWhiteSpace(balance[i]))
+                {
+                    decimal.TryParse(balance[i], out auxArray[j]);
+                    j++;
+                }
+                
+            }
+
+            return auxArray[..j];
+        }
+
+
+        /// <summary>
+        /// Calculte the max balance
+        /// </summary>
+        /// <param name="array"></param>
+        /// <returns></returns>
+        private static decimal MaxBalance(decimal[] array)
+        {
+            if (IsDecArrayNullOrEmpty(array))
+            {
+                return decimal.MinValue;
+            }
+
+            decimal max = decimal.MinValue;
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                if (max < array[i])
+                {
+                    max = array[i];
+                }
+            }
+            return max;
+        }
+        /// <summary>
+        /// Calculate min balance
+        /// </summary>
+        /// <param name="array"></param>
+        /// <returns></returns>
+        private static decimal MinBalance(decimal[] array)
+        {
+            if (IsDecArrayNullOrEmpty(array))
+            {
+                return decimal.MaxValue;
+            }
+
+            decimal min = decimal.MaxValue;
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                if (min > array[i])
+                {
+                    min = array[i];
+                }
+            }
+            return min;
+        }
+
+        /// <summary>
+        /// calculate the balance difference array: Balance[i+1] - Balance[i] with i < Balance.Legth - 1
+        /// </summary>
+        /// <param name="array"></param>
+        /// <returns>an array with the differences</returns>
+        private static decimal[] GetDiff(decimal[] array)
+        {
+            if (IsDecArrayNullOrEmpty(array))
+            {
+                return array;
+            }
+
+            if (array.Length == 1)
+            {
+                return new decimal[] { 0 };
+            }
+
+            decimal[] auxArray = new decimal[array.Length - 1];
+
+            for (int i = 0; i < array.Length - 1; i++)
+            {
+                auxArray[i] = array[i+1] - array[i];
+            }
+
+            return auxArray;
+        }
+
+        /// <summary>
+        /// Changes the Strinbuilder instance by Replacing the last comma with the string " and" in a string with values separated by comma.
+        /// </summary>
+        /// <param name="sb"></param>
+        private static void ReplaceLastCommaWithAnd(StringBuilder sb) 
+        {
+            int lastComma = sb.ToString().LastIndexOf(",");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5069037)

Reply:
Ok, do I need to define locale in this case? I'm replacing the last comma in a list of names by an "and"